ops.cr

```diff
@@ -3,12 +3,15 @@ require "colorize"
 require "ops"
 
 def usage
-  STDERR.puts "Usage: ops [-f <filename>] <action>".colorize(:red)
+  STDERR.puts Builtins::Help.usage.colorize(:red)
```

Comment:
Might as well move the `colorize` into `Help#usage`, since it's already colorizing some of the string itself.

Reply:
Actually, this isn't output in red anyway, since the red colour will be set right before `Help#usage` sticks its own colour in.

Reply:
Oh wait, what you mean is that it's not even colourizing it the way it was meant to. I'll fix that.

Reply:
I keep forgetting that `colorize` only does so from the first sighting of uncoloured text until the next sighting of an ANSI colour code (_or none at all_). I always found that a bit odd.